tests/e2e/helpers/base.js

```diff
@@ -104,16 +116,18 @@ export function runBaseTests (br) {
   test('2up mode - Clicking `Previous page` changes the page', async t => {
     const { nav, BRcontainer} = br;
 
+    const goPrevious = await isRightToLeft() ? nav.desktop.goNext : nav.desktop.goPrevious;
```

Comment:
These two ternaries are already used a few times, and I suspect we'll need them more often. Can we abstract this to methods that allow us to request the next or previous elements? Then internally it can determine whether to send the left or right direction flip button.

Reply:
yes, absolutely. and i think we need to rename the nav buttons (ex. `goNext` => `goRight`) as well in order to make this happen.
i'd like to follow that work behind this PR so we can properly do it and won't balloon this task out of scope.

i do think it is absolutely necessary to do as well,

Reply:
hey @shaneriley lmk if we are still ok with moving that work to its own separate PR - i'd like to get this into master branch before embarking so i don't have to fork.

Reply:
Yeah, we can defer to a separate task.